### DISCUSSION TITLE
Mailgun as SMTP service?

### DISCUSSION THREAD

**Discussion by satonotdead** [lukevella/rallly#640] [Ideas]
We don't like to use our server to mail handling because we are trusting into a good and reliable service like Mailgun.

Is it possible to use it with Rally? Could be implemented?

Thanks for your work! 🏄

**Comment by lukevella** [comment]
Hi, yes it certainly could be implemented. In fact, I recently added the ability to use different transports because I wanted to switch to using the Amazon SES API rather than SMTP for rallly.co. Adding more transports is a relatively trivial task but it requires documentation and also to ensure that support for these transports is maintained once it is published, which is why I'm not keen to rush into it. That said, if someone want to pick this up, I'd be open to a pull request for it. I'll move this to discussions and I'll create an issue with more details on how this can be implemented.
